### ISSUE TITLE
[Screen Reader - Makecode-Documentation] Narrator focus is navigating to hidden element in scan mode and reading it as 'image'

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2803]
**User Experience:**
If Narrator in scan mode is navigating to hidden element and reading as image then user with visual disabilities may get misguided to understand the behavior of the controls

**Test Environment:**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge

**Repro Steps**
1-Navigate to url:-https://makecode.microbit.org/device/error-codes
2-Navigate to footer section
3- launch narrator change to scan mode by (caps+ space key)
4-Verify if screen reader moving to any hidden element or not

**Actual Result:**
Narrator in Scan mode is navigating to hidden element and reading as image

**Expected Result:**
In scan mode Narrator focus should not navigate to hidden element

**Note:**
In item mode narrator is navigating to hidden element and reading as image

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b24201d1-2276-4471-8183-e1e5353031ad}

**Refer Attachment:**
[MAS4.2.10_Footer_FocusMovingHiddenElement.zip](https://github.com/Microsoft/pxt/files/1248729/MAS4.2.10_Footer_FocusMovingHiddenElement.zip)

Latest Video:
[Narrator reads hidden element image.zip](https://github.com/Microsoft/pxt/files/1298769/Narrator.reads.hidden.element.image.zip)

**Comment by veler** [comment]
Can't reproduce this issue. Doing those keyboard combination doesn't work. I'm on Windows 10 RS2.

**Comment by v-mesala** [comment]
We have verified the issue OS: RS2 Version 1703(OS Build 15063.483)Win 10, and the issue still repro in current environment, Launch narrator and please use Caps+ space key for scan mode, and navigate by up\down key. Please refer to the video to get more clear view about the issue.

**Comment by v-mesala** [comment]
Verified on URL:https://makecode.microbit.org/device/error-codes and issue is not fixed now. Hence reactivating the bug.

**Comment by veler** [comment]
I cannot reproduce the issue in the footer anymore. It focus on the footer and describes it as the footer (which is normal and was not the case before), but does not have a hidden narrator focus anymore after the Microsoft logo.
Can you please take a new video?

@abchatra 

![capture](https://user-images.githubusercontent.com/3747805/30339738-d3b5f538-97a4-11e7-9d0f-0fec59c0920a.PNG)

**Comment by v-mesala** [comment]
@abchatra : I have added latest video for the issue. In Scan mode, Narrator reads hidden element "Image". 
Prerequisites: Start Scan mode using Caps + Space key and then navigate using down arrow key

**Comment by v-mesala** [comment]
@abchatra added latest video
[Narrator reads hidden element image.zip](https://github.com/Microsoft/pxt/files/1298797/Narrator.reads.hidden.element.image.zip)

**Comment by samelhusseini** [comment]
Fix should be live, please verify.

**Comment by v-swasha** [comment]
Bug is verified on the URL: https://makecode.microbit.org/projects and issue is fixed as now Narrator focus is not navigating to hidden element in scan mode and reading it as 'image' 
Hence closed the bug and please Add close tag as i dont have access to add tag.
